client/webserver/site/src/js/doc.js

```diff
@@ -120,8 +142,13 @@ export default class Doc {
     return x.toLocaleString('en-us', coinValueSpecs)
   }
 
-  /* logoPath creates a path to a png logo for the specified ticker symbol. */
+  /*
+   * logoPath creates a path to a png logo for the specified ticker symbol. If
+   * the symbol is not a supported asset, the generic letter logo will be
+   * requested instead.
+   */
   static logoPath (symbol) {
+    if (BipSymbols.indexOf(symbol) === -1) symbol = symbol.substring(0, 1)
     return `/img/coins/${symbol}.png`
```

Comment:
Would you mind recompressing these png files with oxipng (`oxipng -o max -s *.png`) or optipng (`optipng -strip all -o7 *.png`)?  There's not really a bandwidth issue, but they compress a lot more so might as well.  pngcrush or pngquant still get the job done if you already have those already, but oxipng is the best and pngcrush can't strip metadata.